src/Dropzone/assets/src/controller.ts

```diff
@@ -117,6 +149,7 @@ export default class extends Controller {
             this.inputTarget.style.display = 'none';
             this.placeholderTarget.style.display = 'none';
             this.previewTarget.style.display = 'block';
+            this.containerTarget.classList.add('dropzone-on-drag-leave');
```

Comment:
Not sur about these class names 🤷‍♂️

Reply:
hum, the goal is to allow style on drag leave / drag enter (e.g change color, transition...). I thought it was the most understable name :D but if you have another suggestion no problem ;-)

Reply:
> I thought it was the most understable name :D

Name maybe, but code not so... Did I miss something, or  theses classes are never removed ? If so the animation would only work once.

Reply:
fixed